.travis.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     script:
       # Sync the chain first. It will time out after 30 minutes. Used network: Rinkeby.
       - make statusgo
-      - ./build/bin/statusd -datadir=.ethereumtest/Rinkeby -les -networkid=4 -sync-and-exit=45 -log=WARN -standalone=false -discovery=false
+      - ./build/bin/statusd -datadir=.ethereumtest/Rinkeby -les -networkid=4 -sync-and-exit=45 -log=WARN -standalone=false -discovery=false -ntp-sync-enabled=false
```

Comment:
can it be shorter? tbh i just hate those 2-3 dash flags
anyone against `-ntp` ?

Reply:
sure